src/app/base/components/StatusBar/StatusBar.test.tsx

```diff
@@ -208,3 +210,50 @@ it("displays last image sync timestamp for a rack or region+rack controller", ()
     `Last image sync: ${controller.last_image_sync}`
   );
 });
+
+it("displays the feedback link when analytics enabled and not in development environment", () => {
+  process.env = { ...originalEnv, NODE_ENV: "production" };
```

Comment:
Do these changes to `process.env` persist beyond these tests? I would suggest reverting it after each test in this suite is run to prevent it from causing problems elsewhere